src/components/Articles.jsx

```diff
@@ -109,28 +107,10 @@ function ArticlesList() {
 								{article.title}
 							</a>
 						</h3>
-						<p className="mt-5 line-clamp-3 text-sm leading-6 text-gray-600">
-							description
+						<p className="text-gray-600">
+							Author: <a href="#">{article.author}</a>
```

Comment:
When I click on an article, I get a blank page. It seems like your link is navigating me to a route that doesn't exist. Please change the route in your link to the correct one and then resubmit this one